settings.py

```diff
@@ -76,7 +77,11 @@ def get_setting(setting):
     elif setting in strings:
         return str(utils.ADDON.getSetting(setting))
     elif setting in paths:
-        return translatePath(utils.ADDON.getSetting(setting).encode("utf-8"))
+        try: 
```

Comment:
This is part of your previous pull request. I have already changed the code for this part, so please revert to that solution and include it in your updated commit so that this no longer is part of the pull request.